pkg/datapath/types/loader.go

```diff
@@ -25,6 +27,9 @@ type Loader interface {
 	Reinitialize(ctx context.Context, o BaseProgramOwner, tunnelConfig tunnel.Config, deviceMTU int, iptMgr IptablesManager, p Proxy) error
 	HostDatapathInitialized() <-chan struct{}
 	DeviceHasTCProgramLoaded(hostInterface string, checkEgress bool) (bool, error)
+	ELFSubstitutions(ep Endpoint) (map[string]uint64, map[string]string)
```

Comment:
Could this type just be an alias for `loader.Loader`?

Reply:
No we can't, this type is duplicated to avoid an import cycle. One day, we will be able to remove it, but not yet.